Source/WebCore/css/calc/CSSCalcTree.h

```diff
@@ -728,20 +735,59 @@ struct Progress {
     using Base = Calculation::Progress;
     static constexpr auto id = CSSValueProgress;
 
-    // <progress()> = progress( <calc-sum> from <calc-sum> to <calc-sum> )
+    // <progress()> = progress( <calc-sum>, <calc-sum>, <calc-sum> )
     //     - INPUT: "consistent" <number>, <dimension>, or <percentage>
     //     - OUTPUT: <number> "made consistent"
     static constexpr auto input = AllowedTypes::Any;
     static constexpr auto merge = MergePolicy::Consistent;
     static constexpr auto output = OutputTransform::NumberMadeConsistent;
 
-    Child progress;
-    Child from;
-    Child to;
+    Child value;
+    Child start;
+    Child end;
 
     bool operator==(const Progress&) const = default;
 };
 
+struct MediaProgress {
+    WTF_MAKE_TZONE_ALLOCATED(MediaProgress);
```

Comment:
Don’t we have a `STRUCT` version of this macro that leaves us in public so we don’t have to write public out afterward?

Reply:
WTF_MAKE_STRUCT_TZONE_ALLOCATED!